docs/en/backup-restore.rst

```diff
@@ -2,56 +2,169 @@
+- Add the signed backup JWT as an entry to the backup file. 
+- Encrypt the backup file using the provided key phrases. 
+
+.. note::
+
+  The Backup JWT MAY contain transaction history for each credential entry within the ``credentials_backup`` claim.
+
+  
+
+**Step 5**: The User will be prompted to select the storage for securely storing the backup file based on his preference. This can range from native storage to external storage (e.g., cloud storage, usb, etc.). 
+
+**Step 6**: Considering the native storage as the preferred choice, the file will be stored on the User device.
+
+A non-normative example of the backup JWT is as the following:
+ 
+ .. code-block::
+  
+  { 
+    "alg": "ES256",   
+    "kid": "pDV1n6WNT4qFx2JbTv7S66WE55BkmHNyuEKO4JctiPw",
+    "typ": "wallet_instance_backup+jwt",    
+  }
+  .
+  {
+    "timestamp":"2024-12-13T16:35:06+01:00",
+    "wallet_provider_id":"https://wallet-provider.example.org/",
+    "wallet_instance_version":"v1.0",
+    "wallet_attestation":"eyJhbGciOiJFUzI1NiIsImVVfQz.eyJpc3MiOiAiaH...LCAibmJ",
+    "credentials_backup":[
+     {  
+        "iss": "https://issuer.example.org/v1.0/mdl",
+        "credential_configuration_id":"org.iso.18013-5.1.mDL"
+     },
+     {  
+        "iss": "https://eaa-provider.example.org/",
+        "credential_configuration_id":"EuropeanDisabilityCard"
+     },
+   ]
+  }
+
+The JOSE header of the backup JWT MUST contain the following REQUIRED parameters:
+
+.. list-table::
+    :widths: 20 60 20
+    :header-rows: 1
+
+    * - **JOSE header**
+      - **Description**
+      - **Reference**
+    * - **alg**
+      - A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST be one of the supported algorithms listed in the Section `Cryptographic Algorithms <algorithms.html>`_ and MUST NOT be set to ``none`` or any symmetric algorithm (MAC) identifier.
+      - :rfc:`7516#section-4.1.1`.
+    * - **kid**
+      -  Unique identifier of the ``jwk`` inside the ``cnf`` claim of Wallet Instance as base64url-encoded JWK Thumbprint value.
+      - :rfc:`7638#section_3`.
+    * - **typ**
+      -  It MUST be set to ``wallet-instance-backup+jwt``
+      - N/A
+
+The body of backup JWT contains the following REQUIRED claims:
+
+.. list-table::
+    :widths: 20 60 
+    :header-rows: 1
+
+    * - **Claim**
+      - **Description**
+    * - **timestamp**
+      - UNIX timestamp with the time of backup file creation. This value is updated every time a new credential entry is added to the backup file.
+    * - **wallet_provider_id**
+      - It MUST be set to the identifier of the Wallet Provider.
+    * - **wallet_instance_version**
+      - It MUST be set to  the version of the Wallet Solution that has been backed up.
+    * - **wallet_attestation**
```

Comment:
IA brings wallet unit attestation

Reply:
For now, let's stay aligned with the technical rules. As soon as we introduce the Wallet Unit Attestation, we will also update this paragraph, otherwise we create inconsistency.